value-streams-as-code.md

```diff
@@ -0,0 +1,106 @@
+3. Next, click on the the 3 vertical dots next to the ***Edit*** button and select **Edit JSON** from the menu <p><img src="img/code/edit_json.png" width=800/>
+4. Delete the **Value Stream JSON** content and replace with the following, replacing all occurences of **{your GitHub username}**, both for the `master` value and the `job` value, in the JSON with your GitHub username and ***{your GitHub Org name}*** with the GitHub Organization you are using for this workshop:
+```
+{
+	"phases": [
+		{
+			"id": "TSlZ0sqDuh",
+			"name": "Build",
+			"gates": [
+				{
+					"id": "5eMvajavv",
+					"name": "API Dev",
+					"master": "https://workshop.cb-sa.io/teams-{your GitHub username}/",
+					"job": "{your GitHub username}/{your GitHub Org name}/helloapp-api/development",
+					"feeds": "v-4fcgpTY",
+					"fanout": []
+				},
+				{
+					"id": "ZF5Ou-7vv",
+					"name": "UI Dev ",
+					"master": "https://workshop.cb-sa.io/teams-{your GitHub username}/",
+					"job": "{your GitHub username}/{your GitHub Org name}/helloapp-nodejs/development",
+					"feeds": "release",
+					"fanout": []
+				}
+			]
+		},
+		{
+			"id": "QLggEt9hI",
+			"name": "Test",
+			"gates": [
+				{
+					"id": "v-4fcgpTY",
+					"name": "API Test",
+					"master": "https://workshop.cb-sa.io/teams-{your GitHub username}/",
+					"job": "{your GitHub username}/{your GitHub Org name}/helloapp-api/test",
+					"feeds": "yiK5MUR5Q",
+					"fanout": []
+				}
+			]
+		},
+		{
+			"id": "ExwDnpexcA",
+			"name": "Deploy API",
+			"gates": [
+				{
+					"id": "yiK5MUR5Q",
+					"name": "API Master",
+					"master": "https://workshop.cb-sa.io/teams-{your GitHub username}/",
+					"job": "{your GitHub username}/{your GitHub Org name}/helloapp-api/master",
+					"feeds": "release",
+					"fanout": [],
+					"type": "deployment"
+				}
+			]
+		},
+		{
+			"id": "8THbC9EXqX",
+			"name": "Deploy App",
+			"gates": [
+				{
+					"id": "release",
+					"name": "UI Master",
+					"master": "https://workshop.cb-sa.io/teams-{your GitHub username}/",
+					"job": "{your GitHub username}/{your GitHub Org name}/helloapp-nodejs/master",
+					"fanout": [],
+					"type": "deployment",
+					"feeds": null
+				}
+			]
+		}
+	]
+}
+```
+5. Click the **Save changes** button <p><img src="img/code/save_json.png" width=800/>
+6. You will have a new Value Stream <p><img src="img/code/microservice_vs.png" width=800/>.
+
+### The **helloapp-nodejs** Application
+
+The **helloapp-nodejs** application is a simple NodeJS app that serves a simple index page with a message. 
+
+If you navigate to **helloapp-nodejs** in your Jenkins master, you'll notice that there are two stages, a `Web Tests` stage used to test the NodeJS app using a selenium sandbox which runs on the `development` branch and a `Build and Push Image` stage that runs on `master`. <p><img src="img/code/hellonodejs_dev_branch.png" width=800/>
```

Comment:
"using a selenium sandbox" it actually uses Testcafe - so maybe replace with: 

using a [TestCafe](https://devexpress.github.io/testcafe/) container

Reply:
Updated.